packages/bull-vault/test/integration-test/FlashBullTestFork.t.sol

```diff
@@ -216,20 +467,31 @@ contract FlashBullTestFork is Test {
             IEulerEToken(eToken).balanceOfUnderlying(address(bullStrategy)),
             "Bull ETH in leverage amount mismatch"
         );
-        assertEq(userBullBalanceBefore.sub(bullToRedeem), bullStrategy.balanceOf(user1), "User1 bull balance mismatch");
         assertEq(
-            crabBalanceBefore.sub(crabToRedeem), crabV2.balanceOf(address(bullStrategy)), "Bull crab balance mismatch"
+            userBullBalanceBefore.sub(bullToRedeem),
+            bullStrategy.balanceOf(user1),
+            "User1 bull balance mismatch"
+        );
+        assertEq(
```

Comment:
We should be able to get an exact value for this if we use uniswap quoter contract get exact eth out for squeeth and usdc. Same for flashdeposit.